cluster-install/install.sh

```diff
@@ -493,6 +498,21 @@ get_limits_list_fancy() {
 #----------------------------------------------------------------------
 #-- Util functions
 #----------------------------------------------------------------------
+get_captured_err() {
+    if [ -f "$__ERR_FILE" ]; then
+        cat "$__ERR_FILE"
+        rm -f "$__ERR_FILE" &> /dev/null
+    else
+        echo ""
+    fi
+}
+
+debug_is_on() {
```

Comment:
For readability/usability/consistency, since there were a few places where we did this check.